src/connect.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo 'connect 5C:BA:37:26:6D:9A\nquit' | bluetoothctl
```

Comment:
This still needs the sudo. bluetoothctl will not be able to register the agent without superuser rights.